integration/sql/iface-py/script/build-macos.sh

```diff
@@ -7,35 +7,15 @@
 # It's assumed that it will be run on MacOS
 set -e
 
-echo "Installing homebrew"
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-
-export HOMEBREW_NO_AUTO_UPDATE=1
-export HOMEBREW_NO_INSTALL_CLEANUP=1
-
-echo "Installing Python 3.7"
-brew install python@3.7
-
-which python
-which python3
+if [ -x "$(command -v /usr/local/bin/python3.7)" ]; then
```

Comment:
We need 3.7 because we support 3.7 - if it's compiled with 3.7 API then it will work on subsequent versions.

It's installed in this dir in CI, but it's not required for it to be there in local env. Instead of forcing it to be installed when doing this locally, it just uses Python what you have.